pkg/kubelet/cm/topologymanager/policy_preferred.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2015 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Nit: I'm not sure we need to update the copyright when editing an already existing k8s file.

Reply:
this file was added this week

Reply:
:+1: